src/test/java/org/opensearch/security/dlic/rest/api/TenantsActionTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.dlic.rest.api;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import org.apache.hc.core5.http.HttpStatus;
```

Comment:
Switch to `import org.apache.http.HttpStatus;`, core5 version isn't needed.

Reply:
Note; I've filed [1] to have this be caught by automation

- [1] https://github.com/opensearch-project/security/issues/3581

Reply:
@peternied Should `org.apache.http.HttpStatus` be used in the integrationTest package to make backporting to 2.x easier? The main branch has already upgraded to org.apache.HttpComponents v5 and v4 is used in 2.x. `main` still has a dependency on v4 because its needed by OpenSAML. See details in https://github.com/opensearch-project/security/issues/2932

Reply:
Let's update the import to `org.apache.http.HttpStatus` which will ensure that the auto backport to 2.x will not have merge conflicts around the package name for these classes.